src/python/pants/ivy/ivy.py

```diff
@@ -59,7 +59,6 @@ def ivy_resolution_cache_dir(self):
     """Returns the ivy cache dir used by this `Ivy` instance."""
     return self._ivy_resolution_cache_dir
 
-  @property
```

Comment:
Is this necessary for typechecking, or is it a style change? I'm very fine with it either way.

Reply:
Necessary for type checking. Apparently it's bad to use `@property` and `@contextmanager` on the same function. MyPy complains about doing that.